cmd/kube-apiserver/app/server.go

```diff
@@ -210,9 +212,16 @@ func Run(s *options.APIServer) error {
 				installSSH = instances.AddSSHKeyToAllInstances
 			}
 		}
-
+		if ports.KubeletReadOnlyPort <= 0 {
```

Comment:
Also, the healthcheck below isn't using the read-only port, so I'm not sure why we care about it being set.